debug/images.go

```diff
@@ -0,0 +1,84 @@
+// Copyright © 2021 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package debug
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+type ImagesManagement interface {
+	ShowDefaultImages() error
+	GetDefaultImage() (string, error)
+}
+
+const DefaultSealerRegistryURL = "registry.cn-qingdao.aliyuncs.com/sealer-apps/"
+
+var ShowImagesCMD = &cobra.Command{
+	Use:   "show-images",
```

Comment:
I am wondering if  just `images` is OK, rather than `show-images`.

Reply:
From the semantic, I think the `sealer debug show-images` are clearer to express what the command do. When using `sealer debug images`, it looks more like debugging the images. So, I used images in the beginning, but then I changed it to show-images.